extension.json

```diff
@@ -10,5 +10,8 @@
   ],
   "documentationUrl": {
     "en": "https://github.com/open-contracting/ocds_charges_extension"
-  }
+  },
+  "codelists":[
```

Comment:
Can you ensure consistent whitespace style? (space after colon, two spaces per indentation level)

Reply:
Sure. Is there a warning we can easily add to the maintenance scripts that might check our preferred indentation has been applied?

Reply:
When I was originally testing all JSON files, there were too many errors, but I'll try scoping it to specific filenames, or extensions only.

Reply:
If I do that, I'll have to fix some files in other extensions myself anyway, so this is ready to merge.